bin/process_ctss.sh

```diff
@@ -19,28 +19,14 @@ cat $FILES > ctss_all
 
 
 #  "Sorting pooled ctss"
- sort -k 1,1 -k 2,2n -k 3,3n -k 6,6 ctss_all > ctss_all.Sorted
+sort -k 1,1 -k 2,2n -k 3,3n -k 6,6 ctss_all > ctss_all.Sorted
```

Comment:
This is a little against nf-core design practice (I think) - any reason why you wouldn't expand these steps out into the pipeline? I get if this was all piped together, but you are generating several intermediate files all in one module in a custom script - it doesn't feel right.